tests/models/test_pyro.py

```diff
@@ -156,9 +207,19 @@ def test_pyro_bayesian_regression(save_path):
 def test_pyro_bayesian_regression_jit():
     use_gpu = int(torch.cuda.is_available())
     adata = synthetic_iid()
+    # add index for each cell (provided to pyro plate for correct minibatching)
```

Comment:
we should comment that this isn't strictly necessary for this particular test case model as it only has global latent variables. In other words the old code worked too.

Reply:
I extended this model to (optionally) has cell-specific latent variables - that feature needs to be tested, right?

Reply:
This is necessary to add even in those tests because the model now has ind_x argument